notebooks/loader_testing.ipynb

```diff
@@ -0,0 +1,231 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "70b1ee9b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"src\")  #You might have to modify this according to the directory structure\n",
```

Comment:
I think we need to create an __init__.py file in the src folder to turn it into a module to allow for this. This path should also be structured, see the other comment for how to do that.